docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -90,6 +90,19 @@ For more information, refer to {agent-issue}2103[#2103].
 Ensure that the application used to install {agent} (for example, the Terminal or custom package) has Full Disk Access before running `sudo ./elastic-agent install`.
 ====
 
+[[known-issue-issue-2342-8.6.2]]
+.{agent} upgrades scheduled for a future time do not run.
+[%collapsible]
+====
+*Details* +
+A known issue in {agent} may prevent upgrades scheduled to execute at a later time from running.
+For more information refer to {agent-issue}2342[#2342].
+
+*Impact* +
+{kib} may show an agent as being stuck with the `Updating` status.
+If the scheduled start time has passed, you may force the agent to run by sending it any action, such as a change to the policy or the log level.
```

Comment:
Does manually triggering the upgrade again work? Should that be the first suggested work around since this primarily applies to upgrades?

Is there any chance the upgrade would happen twice in this situation, once from the pending scheduled action and once from the non-scheduled upgrade action?

Reply:
Manually triggering does not work - kibana will say the agent is not upgradable

Reply:
Let's document that explicitly in the known issue to avoid getting future questions about this then.